controllers/integrationpipeline/integrationpipeline_controller.go

```diff
@@ -19,7 +19,6 @@ package integrationpipeline
 import (
```

Comment:
should we also remove this?
https://github.com/redhat-appstudio/integration-service/blob/613c0c99f8e84504025195bf95baada1cb7b9862/controllers/integrationpipeline/integrationpipeline_controller.go#L122-L124

Reply:
Good call, I'll move that to a more appropriate place since we do need it for build pipelineRuns.

Reply:
Moved it to the buildpipeline controller instead.

Reply:
Thanks!